cli/src/commands/bookmark/list.rs

```diff
@@ -354,6 +364,28 @@ impl SortKey {
     }
 }
 
+struct SortKeys(Vec<SortKey>);
+
+impl TryFrom<ConfigValue> for SortKeys {
+    type Error = String;
+
+    fn try_from(value: ConfigValue) -> Result<Self, Self::Error> {
```

Comment:
nit: no need to add newtype for parsing. Just pass `parse_sort_keys()` function to `get_value_with()`.

We can also use `serde::Deserialize` to `get()` `Vec<SortKeys>` directly, but it might be tedious to maintain both clap's `name` and `serde(rename = ..)`.